api/admin/cli/config.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Actually cli admin should receive config as a ferrum, therefore it must use ferrum functionality and it will-be more user friendly

Reply:
The `Server` section will be checked, which is not required for the cli. And fail with an error
Example section Server:
```json
    "server": {
        "schema": "http",
        "address": "localhost",
        "port": 8182,
        "secret_file": "./keyfile"
    },
```

Reply:
Even it does not need , it is not user friendly to have 2 different config for app itself and for administration, such aapoach is wrong